app/frontend/src/views/Home.vue

```diff
@@ -1,12 +1,149 @@
+    <v-simple-table>
+      <template v-slot:default>
+        <thead>
+          <tr>
+            <th class="text-left">Description</th>
+            <th class="text-left">Type</th>
+            <th class="text-left">Template</th>
+            <th class="text-left">Contexts</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="(item, index) in examples" :key="`item-${index}`">
+            <td>{{ item.description }}</td>
+            <td>{{ item.type }}</td>
+            <td>
+              <v-btn
+                :color="item.template.color"
+                class="btn-template-download"
+                download
+                :href="item.template.link"
+                outlined
+                small
+              >
+                <v-icon left>{{ item.template.icon }}</v-icon>
+                {{ item.template.link.replace(/^.*\//, '') }}
+              </v-btn>
+            </td>
+            <td>
+              <v-btn
+                :color="item.contexts.color"
+                class="btn-contexts-download"
+                download
+                :href="item.contexts.link"
+                outlined
+                small
+              >
+                <v-icon left>mdi-download</v-icon>
+                {{ item.contexts.link.replace(/^.*\//, '') }}
+              </v-btn>
+            </td>
+          </tr>
+        </tbody>
+      </template>
+    </v-simple-table>
   </v-container>
 </template>
 
 <script>
 export default {
-  name: 'home'
+  name: 'home',
+  data() {
+    return {
+      examples: [
+        {
+          contexts: {
+            color: 'teal',
+            link: './examples/contexts_dgrsc.json',
+          },
+          description: 'Information Sharing Agreement',
+          template: {
+            color: 'blue',
+            icon: 'mdi-file-word',
+            link: './examples/template_information_sharing_agreement.docx',
+          },
+          type: 'Microsoft Word Document (.docx)'
+        },
+        {
+          contexts: {
+            color: 'teal',
+            link: './examples/contexts_dgrsc.json',
+          },
+          description: 'Information Sharing Agreement',
+          template: {
+            color: 'brown',
+            icon: 'mdi-file-document',
+            link: './examples/template_information_sharing_agreement.odt',
+          },
+          type: 'OpenDocument Text (.odt)'
+        },
+        {
+          contexts: {
+            color: 'teal',
+            link: './examples/contexts_dgrsc.json',
+          },
+          description: 'Sample PowerPoint Presentation',
+          template: {
+            color: 'orange',
+            icon: 'mdi-file-powerpoint',
+            link: './examples/template_powerpoint.pptx',
+          },
+          type: 'Microsoft PowerPoint Presentation (.pptx)'
+        },
+        {
+          contexts: {
+            color: 'red',
```

Comment:
Would change to black

Reply:
Made all JSON files teal for consistency.